api/docker/php/docker-entrypoint.sh

```diff
@@ -14,7 +14,7 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'bin/console' ]; then
 	if [ "$APP_ENV" != 'prod' ]; then
 		composer install --prefer-dist --no-progress --no-suggest --no-interaction
 		>&2 echo "Waiting for Postgres to be ready..."
-		until bin/console doctrine:query:sql "SELECT 1" --quiet; do
+		until pg_isready --timeout=0 --dbname="${DATABASE_URL}"; do
```

Comment:
Even if we prefer using a Postgres database for performances reasons, this command definitively connects this entrypoint to the database system & driver. I don't find it very useful, especially if the final user wants to use a different system.
WDYT @dunglas?

Reply:
There is a better way. Thanks to the Sylius peeps who used it: https://github.com/Sylius/Sylius-Standard/blob/943125142f0267ac49903efeaab85e24de08424d/docker/php/docker-entrypoint.sh#L20-L23

Let's adopt the same method as well.

Reply:
It seems the same, but what is the purpose of using ` >/dev/null 2>&1` instead of `--quiet`?

Reply:
We used this Doctrine command long ago, but I can't remember why we switched. `pg_isready` looks more reliable anyway.

> Even if we prefer using a Postgres database for performances reasons, this command definitively connects this entrypoint to the database system & driver. I don't find it very useful, especially if the final user wants to use a different system.

It's the same for the PDO driver and so on. The entrypoint is part of the infrastructure, to me it's ok and intended to couple it with the services we use.

Reply:
Anyway, please update `api-platform/api-platform` first, and the demo after by copying it.